content/en/docs/application-store/external-apps/deploy-tidb.md

```diff
@@ -0,0 +1,217 @@
+
+   ```bash
+   kubectl apply -f https://raw.githubusercontent.com/pingcap/tidb-operator/v1.1.6/manifests/crd.yaml
+   ```
+
+2. You can see the expected output as below:
+
+   ```bash
+   customresourcedefinition.apiextensions.k8s.io/tidbclusters.pingcap.com created
+   customresourcedefinition.apiextensions.k8s.io/backups.pingcap.com created
+   customresourcedefinition.apiextensions.k8s.io/restores.pingcap.com created
+   customresourcedefinition.apiextensions.k8s.io/backupschedules.pingcap.com created
+   customresourcedefinition.apiextensions.k8s.io/tidbmonitors.pingcap.com created
+   customresourcedefinition.apiextensions.k8s.io/tidbinitializers.pingcap.com created
+   customresourcedefinition.apiextensions.k8s.io/tidbclusterautoscalers.pingcap.com created
+   ```
+
+### Step 2: Add the app repository
+
+1. Log out of KubeSphere and log back in as `ws-admin`. In your workspace, go to **App Repos** under **Apps Management**, and then click **Add Repo**.
+
+   ![add-repo](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/add-repo.PNG)
+
+2. In the dialog that appears, enter `pingcap` for the app repository name and `https://charts.pingcap.org` for the PingCAP Helm repository URL. Click **Validate** to verify the URL and you will see a green check mark next to the URL if it is available. Click **OK** to continue.
+
+   ![add-pingcap-repo](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/add-pingcap-repo.PNG)
+
+3. Your repository displays in the list after successfully imported to KubeSphere.
+
+   ![added-pingcap-repo](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/added-pingcap-repo.PNG)
+
+### Step 3: Deploy the TiDB Operator
+
+1. Log out of KubeSphere and log back in as `project-regular`. In your project, go to **Applications** under **Application Workloads** and click **Deploy New Application**.
+
+   ![deploy-app](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/deploy-app.PNG)
+
+2. In the dialog that appears, select **From App Templates**.
+
+   ![from-app-templates](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/from-app-templates.PNG)
+
+3. Select `pingcap` from the drop-down list, then click **tidb-operator**.
+
+   ![click-tidb-operator](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/click-tidb-operator.PNG)
+
+   {{< notice note >}}
+
+   This tutorial only demonstrates how to deploy TiDB Operator and a TiDB cluster. You can also deploy other tools based on your needs.
+
+   {{</ notice >}}
+
+4. Select the **Chart Files** tab, you can view the configuration from the console directly or download the default `values.yaml` file by clicking the icon on the upper right corner. Under **Versions**, select a version number from the drop-down list and click **Deploy**.
```

Comment:
Select the **Chart Files** tab
->
On the **Chart Files** tab

on the upper right corner.
->
in the upper right corner.